### PR TITLE
Fixes #10080: Reorganize Directives page interface

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/10080

### CODE REVIEW

File: rudder-web/src/main/webapp/javascript/rudder/rudder.js [hunk 1]
```diff
@@ -653,10 +653,23 @@ function hideFileManager(){
...
+      if(toolbar && $(toolbar).offset()){
+        toolbarHeight = parseFloat($(toolbar).css('height'));
+      }else{
+        toolbarHeight = 0;
+      }
+      var offsetTop = treeOffset.top + 15 + toolbarHeight;
+      maxHeight = 'calc(100vh - '+ offsetTop + 'px)';
+    }
+  }else{
+    maxHeight = "none";
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/javascript/rudder/rudder.js]: You should set maxHeight to "none" by default and do not do the else part

File: rudder-web/src/main/webapp/style/rudder/rudder-menu.css [hunk 2]
```diff
@@ -1050,6 +1058,46 @@ table.tablewidth{
 }
 
 /* --- DIRECTIVE PAGE --- */
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/style/rudder/rudder-menu.css]: I would love to have this in a separate file 

directives.css

File: rudder-web/src/main/webapp/style/rudder/rudder-menu.css [hunk 3]
```diff
@@ -876,7 +876,15 @@ a.sidebar-toggle{
   position:relative;
   opacity:1;
 }
-/* RULE PAGE */
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/style/rudder/rudder-menu.css]: Same here I would love to have it in a separate file, only loaded when needed

**Comment by VinceMacBuche** [comment]
Ap&art the three small comment this looks ok!

**Comment by VinceMacBuche** [comment]
Oops

**Comment by RaphaelGauthier** [comment]
Commit modified

File: rudder-web/src/main/webapp/style/rudder/rudder-directives.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
+/* --- DIRECTIVES PAGE --- */
```
Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/style/rudder/rudder-directives.css]: You need to add the licence file like in rudder.css

Comment (VinceMacBuche) [inline rudder-web/src/main/webapp/style/rudder/rudder-directives.css]: licens lines ...

**Comment by RaphaelGauthier** [comment]
Commit modified

**Pull request merged**